packages/material-ui-lab/src/TimelineContent/TimelineContent.js

```diff
@@ -1,39 +1,67 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { deepmerge } from '@material-ui/utils';
 import { capitalize } from '@material-ui/core/utils';
-import { withStyles } from '@material-ui/core/styles';
+import { experimentalStyled } from '@material-ui/core/styles';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import useThemeProps from '@material-ui/core/styles/useThemeProps';
```

Comment:
More than 1-level deep imports are not allowed. They break the UMD vs. CJS isolation
```suggestion
import { useThemeProps } from '@material-ui/core/styles';
```